source/project-stories/_project_team_single.erb

```diff
@@ -0,0 +1,19 @@
+<section class="project-team">
+  <h2 class="project-team__title">
+    <%= current_page.data.team.title %>
+  </h2>
+
+  <% team_size = current_page.data.team.people.size %>
+
+  <% current_page.data.team.people.each do |person| %>
+    <% if person.link %>
+      <a class="project-team__headshot-container-single project-team__container--<%= team_size %>" href="<%= person.link %>">
+        <img class="project-team__headshot" src="<%= person.image %>" alt="<%= person.full_name %>" title="<%= person.full_name %>">
+      </a>
+    <% else %>
+      <div class="project-team__headshot-container project-team__container--<%= team_size %>">
+        <img class="project-team__headshot" src="<%= person.image %>" alt="<%= person.full_name %>" title="<%= person.full_name %>">
+      </div>
+    <% end %>
+  <% end %>
+</section>
```

Comment:
See [my other comment](https://github.com/unboxed/ubxd_web_refresh/pull/190/files#r71673022). tl;dr we shouldn't introduce this file.